simple/tests/stats/test_data/runner/config/config_driven.json

```diff
@@ -10,6 +10,12 @@
       "importType": "observations",
       "entityType": "Country",
       "provenance": "Provenance1 Name"
+    },
+    "svobs.csv": {
```

Comment:
not using "svobs"? This is more of an internal acronym, maybe something matching the file format config.

Reply:
Done.